src/wallet/wallet.cpp

```diff
@@ -2644,6 +2644,31 @@ bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nC
     return true;
 }
 
+OutputType CWallet::TransactionChangeType(const std::vector<CRecipient>& vecSend)
```

Comment:
Suggestions, for now, keep static and not a `CWallet` member:
```cpp
static OutputType GetChangeTypeForRecipients(const std::vector<CRecipient>& recipients)
{
```
And below:
```cpp
const OutputType change_type = GetChangeTypeForRecipients(vecSend);
```
*don't mind the function name.*

Reply:
What's the advantage of making it static? Especially if the longer term aim is to specify launch options for individual wallets?

Reply:
I just think it's too soon to *expose* this in the `CWallet`. Feel free to leave as it is.

Reply:
cc @jnewbery given your last comment. I'll leave it as is, but useful to know what the future direction shouldl be.